src/libs/datatableproviders/EntityServiceDataProvider.ts

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line lodash-fp/use-fp
+import { ValueKeyIteratee } from 'lodash'
+import _ from 'lodash/fp'
+import { Ajax } from 'src/libs/ajax'
+import { DataProvider, DataProviderFeatures, DeleteTable, EntityQuerySortDirection, GetMetadata, GetPage } from 'src/libs/datatableproviders/DataProvider'
+
+
+export class EntityServiceDataProvider implements DataProvider {
+  features: DataProviderFeatures = {
+    enableTsvDownload: true,
+    enableTypeDeletion: true,
+    enableTypeRenaming: true,
+    enableExport: true,
+    enablePointCorrection: true,
+    enableFiltering: true
+  }
+
+  getPage: GetPage = async (signal: any, _workspaceId: string, entityType: string, pageNumber: number,
+    itemsPerPage: number, sortField: string, sortDirection: EntityQuerySortDirection,
+    namespace: string, name: string, snapshotName: string, googleProject: string, activeTextFilter: string, filterOperator: string) => {
+    return await Ajax(signal).Workspaces.workspace(namespace, name)
```

Comment:
`return await` may not be necessary.

https://eslint.org/docs/latest/rules/no-return-await

Reply:
hm - I understand the objective here, but removing the `await` causes other issues:
* eslint now warns about [require-await](https://eslint.org/docs/latest/rules/require-await): Async method 'getPage' has no 'await' expression.
* if I understand correctly, it makes these functions not thenable, which seems like a loss

Do you feel strongly about removing the `await` here?

Reply:
I don't feel strongly about it, since there are reasons to use it as described in https://eslint.org/docs/latest/rules/no-return-await#when-not-to-use-it.

However, 
> eslint now warns about [require-await](https://eslint.org/docs/latest/rules/require-await): Async method 'getPage' has no 'await' expression.
If the `await` is removed, then function the function does not need to be marked as `async`.

> if I understand correctly, it makes these functions not thenable, which seems like a loss
The `Ajax().Workspaces.workspace(...).paginatedEntitiesOfType(...)` call returns a Promise, which is "thenable" and can be awaited. Awaiting here is similar to wrapping that in another Promise.